app/models/walletkey.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const _ = require('lodash');
+
+const walletKeySchema = new mongoose.Schema({
+  masterKey: { type: mongoose.Types.ObjectId },
+  path: { type: mongoose.Types.String },
+  xpub: { type: mongoose.Types.String },
+  userEmail: { type: mongoose.Types.String },
+  notificationUrl: { type: mongoose.Types.String },
+  verificationInfo: { type: mongoose.Types.String },
```

Comment:
Why would verification info be on a per-wallet basis rather than per-customer? Would a customer ever have different verification for different wallets?

Reply:
Check out Keyternal's recovery policy at https://keytern.al/p/recovery/ . They have different verification steps based on wallet value, and I believe these are per-wallet, not per-customer. 

I believe in v1 they were storing some verification notes on each individual Key document (thoguh it's hard to be sure as they've changed so much of the code)